rules/aws/credential_access_aws_iam_assume_role_brute_force.toml

```
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed attempts to assume an AWS IAM role. IAM roles are used to delegate access to users or
+services. An adversary may attempt to enumerate IAM roles in order to determine whether a role exists or not before
```

```suggestion
services. An adversary may attempt to enumerate IAM roles in order to determine if a role exists before
```

(may need a re-`toml-lint`)